finetune/base_models/glove/model.py

```diff
@@ -0,0 +1,43 @@
+import os
+from urllib.parse import urljoin
+
+from finetune.base_models import SourceModel
+from finetune.base_models.glove.encoder import GloveEncoder
+from finetune.base_models.glove.featurizer import glove_featurizer
+from finetune.util.download import GLOVE_BASE_URL, FINETUNE_BASE_FOLDER
+
+class GloveModel(SourceModel):
+    is_bidirectional = False
```

Comment:
Think this is just used for some stuff in model.py re: text generation but not sure.

Reply:
This is also used for the sequence model to control if an extra block is added for the sequence_labeler